.updatecli/bump-opbeans-node.yml

```diff
@@ -0,0 +1,40 @@
+---
+## Workflow to periodically check if there is an available newer APM agent version, e.g.
+## "1.2.3". If so, then update to it.
+name: Bump elastic-apm-node to latest version
+
+sources:
+  elastic-apm-node:
+    name: Get latest elastic-apm-node version from npm registry
+    kind: npm
+    spec:
+      name: elastic-apm-node
+
+conditions:
+  elastic-apm-node-version-check:
+    name: Check if installed version differs
+    kind: shell
+    sourceid: elastic-apm-node
+    spec:
+      command: bash .updatecli/scripts/test-version.sh
+      environments:
+        - name: PATH
```

Comment:
Probably you can get rid of the condition due to `npm install` will not modify the packages file.

Reply:
I just tested it and omitting this part makes the `npm` target run always.  Regardless if it's installed already.